File: Gfg/Sort012.java
class Sort012 {

    // a brute force solution(i guess) to sort all the 0 1 and 2s in the array

    public static void sort012(int a[], int n) {
        int counter0 = 0;
        int counter1 = 0;
        int counter2 = 0;
        for ( int i = 0 ; i < n ; i++ ) {
            if ( a[i] == 0 ) {
                counter0++;
            } else if ( a[i] == 1 ) {
                counter1++;
            } else {
                counter2++;
            }
        }
        for ( int i = 0 ; i < n ; i++ ) {
            if ( counter0 > 0 ) {
                a[i] = 0;
                counter0--;
            } else {
                if ( counter1 > 0 ) {
                    a[i] = 1;
                    counter1--;
                } else {
                    a[i] = 2;
                }
            }
        }
    }
}

File: Algorithms/Gfg/Sort012.py
# i guess its a brute force algorithm to sort the numbers from 0 to 2 in an array
def sort012(array , n):
	counter0 = 0
	counter1 = 0
	counter2 = 0
	for i in range(0,n):
		if array[i] == 0:
			counter0 += 1
		elif array[i] == 1:
			counter1 += 1
		else:
			counter2 += 1

	for i in range(0,n):
		if counter0 > 0:
			array[i] = 0
		else:
			if counter1 > 0:
				array[i] = 1
			else:
				array[i] = 2